### PR TITLE
fix(Icon): alt text and description as defaultProps (a11y/il8n)

### PR DESCRIPTION
**Pull request by hellobrian**
Related to #107

### CODE REVIEW

File: components/Icon.js [hunk 1]
```diff
@@ -12,12 +12,13 @@ const propTypes = {
...
   style: PropTypes.object,
   viewBox: PropTypes.string,
-  width: PropTypes.string,
+  width: PropTypes.string
 };
 
 const defaultProps = {
   fillRule: 'evenodd',
   role: 'img',
+  description: 'Provide a description that will be used as the title'
```
Comment (hellobrian) [inline components/Icon.js]: `description` prop is now in `defaultProps` for `il8n`

File: components/Icon.js [hunk 2]
```diff
@@ -114,13 +113,13 @@ const Icon = ({
...
     width: width || icon.width,
-    ...other,
+    ...other
   };
 
   const svgContent = icon ? svgShapes(icon.svgData) : '';
 
   return (
-    <svg {...props}>
+    <svg {...props} aria-label={description}>
```
Comment (hellobrian) [inline components/Icon.js]: Including `aria-label` with `description` prop to satisfy a11y issue raised by aXe.
According to [StackOverflow](https://stackoverflow.com/a/4756461/2058360):

image

**Comment by shyfruan** [comment]
@hellobrian Can you merge the pr ASAP? 
We need the avt fix in our project. Thank you

**Comment by chrisdhanaraj** [comment]
👍  good by eme

**Comment by iangfleming** [comment]
👍

**Comment by chrisdhanaraj** [comment]
Bluemix [toolchain](https://console-regional.ng.bluemix.net/devops/toolchains/6b8b7163-1ec8-4616-82fa-aa467a17af03 "storybook-toolchain-v2"): [Delivery Pipeline](https://console-regional.ng.bluemix.net/devops/pipelines/b84aaac8-8484-4a1e-902a-ebb526eb7998 "storybook-toolchain-v2") deployed [carbon-components-react](https://github.com/carbon-design-system/carbon-components-react/releases/tag/deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20170802-190905 "deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20170802-190905") to [Bluemix Design System](https://console.bluemix.net/apps/0a253845-97bd-46c9-aa04-a220aeb8ed06?env_id=ibm:yp:us-south "ibm:yp:us-south:chrisdhanaraj_org:Bluemix Design System"), including commit 19d550bb37fd353c25a2a7e463992893f4b38a95

**Pull request merged**